### DISCUSSION TITLE
Use sarama with kerberos enabled

### DISCUSSION THREAD

**Discussion by simonstumpf** [IBM/sarama#2625] [Q&A]
##### Versions

*Please specify real version numbers or git SHAs, not just "Latest" since that changes fairly regularly.*

| Sarama | Kafka | Go |
|--------|-------|----|
|latest  |2.4.1.7  |1.15  |

##### Configuration

What configuration values are you using for Sarama and Kafka?
``` go
```

##### Logs

When filing an issue please provide logs from Sarama and Kafka if at all
possible. You can set `sarama.Logger` to a `log.Logger` to capture Sarama debug
output.

<details><summary>logs: CLICK ME</summary>
<p>

```
```

</p>
</details>

##### Problem Description
Hi All

Sorry for this question, I'm new in Kafka.

I'm trying to connect to my cloudera kafka cluster via Kerberos.
It works fine with the kafka command line tools, but I can't get it to work with sarama.

My jaas.conf looks like this: 
```Client {

  com.sun.security.auth.module.Krb5LoginModule required

  useKeyTab=false

  useTicketCache=true

  renewTicker=true

  debug=true

  principal="PRINCIPAL";

}```

Does anyone knows how I need to configure sarama?

**Comment by ghost** [comment]
Thank you for taking the time to raise this issue. However, it has not had any activity on it in the past 90 days and will be closed in 30 days if no updates occur.
Please check if the master branch has already resolved the issue since it was raised. If you believe the issue is still valid and you would like input from the maintainers then please comment to ask for it to be reviewed.
